src/domain/template/templates-set/templates.set.service.ts

```diff
@@ -106,6 +106,13 @@ export class TemplatesSetService {
         );
       }
     }
+    if (templatesSet.calloutTemplates) {
```

Comment:
I think you need to load the relation in
    const templatesSet = await this.getTemplatesSetOrFail(templatesSetID, {
      relations: {
        authorization: true,
        postTemplates: true,
        whiteboardTemplates: true,
        innovationFlowTemplates: true,
        communityGuidelinesTemplates: true,
      },
    });